### ISSUE TITLE
Makefile PY_BASE_LINK_DIR does not accept value from the environment

### ISSUE THREAD

**Issue by dstndstn** [dstndstn/astrometry.net#142]
PY_BASE_LINK_DIR, unlike some other Makefile variables, does not accept the environment variable as a default; this is a variable whose value needs to be overridden in some cases.

**Comment by dstndstn** [comment]
fixed by b5144ef7

**Comment by zbeekman** [comment]
Documenting Homebrew changes needed here, for future reference:

For the next release, add:

```
ENV["PY_BASE_LINK_DIR"] = libexec/"lib/python#{xy}/site-packages/astrometry"
```

after

```
ENV["PY_BASE_INSTALL_DIR"] = "#{libexec}/lib/python#{xy}/site-packages/astrometry"
```

and remove `bin.install_symlink ...`
